src/components/plots/piePlot.ts

```diff
@@ -67,6 +79,82 @@ export module Plots {
 
       return allPlotData;
     }
+
+    public sectorValue(): ValueScaleBinding<number, D, number>;
+    public sectorValue(sectorValue: number | _Accessor): Plots.Pie<D>;
+    public sectorValue(sectorValue: D | _Accessor, sectorValueScale: Scale<D, number>): Plots.Pie<D>;
+    public sectorValue(sectorValue?: number | _Accessor | D, sectorValueScale?: Scale<D, number>): any {
+      if (sectorValue == null) {
+        return Pie._constructBinding(this._sectorValue, this._sectorValueScale);
+      }
+      this._sectorValue = sectorValue;
+      Pie._replaceScaleBinding(this._sectorValueScale, sectorValueScale, this._renderCallback);
+      this._sectorValueScale = sectorValueScale;
+      this._render();
+      return this;
+    }
+
+    public innerRadius(): ValueScaleBinding<number, D, number>;
+    public innerRadius(innerRadius: number | _Accessor): Plots.Pie<D>;
+    public innerRadius(innerRadius: D | _Accessor, innerRadiusScale: Scale<D, number>): Plots.Pie<D>;
+    public innerRadius(innerRadius?: number | _Accessor | D, innerRadiusScale?: Scale<D, number>): any {
+      if (innerRadius == null) {
+        return Pie._constructBinding(this._innerRadius, this._innerRadiusScale);
+      }
+      this._innerRadius = innerRadius;
+      Pie._replaceScaleBinding(this._innerRadiusScale, innerRadiusScale, this._renderCallback);
+      this._innerRadiusScale = innerRadiusScale;
+      this._render();
+      return this;
+    }
+
+    public outerRadius(): ValueScaleBinding<number, D, number>;
+    public outerRadius(outerRadius: number | _Accessor): Plots.Pie<D>;
+    public outerRadius(outerRadius: D | _Accessor, outerRadiusScale: Scale<D, number>): Plots.Pie<D>;
+    public outerRadius(outerRadius?: number | _Accessor | D, outerRadiusScale?: Scale<D, number>): any {
+      if (outerRadius == null) {
+        return Pie._constructBinding(this._outerRadius, this._outerRadiusScale);
+      }
+      this._outerRadius = outerRadius;
+      Pie._replaceScaleBinding(this._outerRadiusScale, outerRadiusScale, this._renderCallback);
+      this._outerRadiusScale = outerRadiusScale;
+      this._render();
+      return this;
+    }
+
+    public scaledInnerRadiusAccessor(): _Accessor {
+      return Pie._scaledValueAccessor(this._innerRadius, this._innerRadiusScale);
+    }
+
+    public scaledOuterRadiusAccessor(): _Accessor {
+      return Pie._scaledValueAccessor(this._outerRadius, this._outerRadiusScale);
+    }
+
+    public scaledSectorValueAccessor(): _Accessor {
```

Comment:
That doesn't seem correct to me. `Drawer`s are essentially an implementation detail, so we're adding public API points for no good reason except that some internal user needs them. The logic required to generate the `_scaledValueAccessor` isn't that complex either.